sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsublite/PubsubLiteIO.java

```diff
@@ -64,8 +64,8 @@ private PubsubLiteIO() {}
    *     .build()), "read");
    * }</pre>
    */
-  public static Read.Unbounded<SequencedMessage> read(SubscriberOptions options) {
-    return Read.from(new PubsubLiteUnboundedSource(options));
+  public static PTransform<PBegin, PCollection<SequencedMessage>> read(SubscriberOptions options) {
```

Comment:
I don't (it adds to the api surface for an uncommon case), but this can be implemented in userland by:

```
PCollectionList list = PCollectionList.of();
for (SubscriptionPath subscription : subscriptions) {
  list = list.and(p.apply(PubsubLiteIO.read(SubscriberOptions.newBuilder()
      .setSubscriptionPath(subscriptionPath)
      .build()), "read"));
}
return list.apply(Flatten.<SequencedMessage>pCollections())
```

Reply:
The main goal for us to build composite transform is to offer end users simple API to do complex work. I would say it's more convenient for pipeline authors to have `readAll()` API instead of writing more code by their own.

Reply:
Yeah I don't agree. Reading from multiple subscriptions to process the same data is a niche use-case: especially given that we try to commit to not only never breaking API but the internal structure to prevent from breaking dataflow refreshing, I think we should keep the API surface as small as possible. Given how niche this is and how easy it is (5 LOC), I see no benefit in adding noise to the API surface of PubsubLiteIO.

Reply:
I'll let you know when there is a FR on `readAll`